### DISCUSSION TITLE
There is a repo on my Repositories list but not mine

### DISCUSSION THREAD

**Discussion by ghost** [community/community#102664] [Repositories] [answered]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

i never pull codes to the repo, but it always on my repo list, now i want delete it but noway, how can i fix this bug?

(the repo i want to delete is the last one on the list)

![image](https://github.com/community/community/assets/77842352/d450de18-f861-4420-858d-932ce0df190d)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by emad313** [comment]
Do you fork anyone's repo?

If you fork someone's repo, then go to that repository setting and find the delete option from the bottom of the page.

**Comment by tabers77** [comment]
I would suggest checking whether you are a collaborator on the repository. Verify if you are watching or have starred the repository; if so, remove it. Additionally, ensure that you haven't performed any actions associated with the repository.

**Reply by spenserblack** [reply]
Yeah I think it's this. I have a repo in my list that isn't mine, and it's one that I'm a collaborator on.

**Comment by ghost** [comment]
![image](https://github.com/community/community/assets/77842352/1a4e9995-d69f-4da5-bdc4-5a39e19b97b1)
![image](https://github.com/community/community/assets/77842352/3e6a7be3-bdb7-4ee7-a2fb-d1975f8e099a)
and i never add code to the repo ,and theres no setting button here

**Reply by liamvdm2** [reply]
but do you have collaborators on a repo? or are you a collaborator on the repo of that project?

**Reply by ghost** [reply]
![image](https://github.com/community/community/assets/77842352/08240ef2-1ad8-4eb8-ae38-468e62637b95)
but theres none actually

**Reply by spenserblack** [reply]
Do you frequently visit the repository? Some repositories on my list I'm not a collaborator or contributor, but I do frequently visit and respond to issues.

**Reply by ghost** [reply]
long time ago i answer a question in it  but i want delete it  how to do

**Comment by Olayiwolaaa** [comment]
- Check if you are a collaborator on the repository or if it belongs to an organization you're a member of. If so, you may need to ask the repository owner or organization administrator to remove your access.
- If the repository is a fork of another repository, check if you forked it. If you did, you can simply delete your fork.
- If you didn't fork it, someone else might have forked it and added you as a collaborator. You'll need to ask the person who forked it to remove your access.
- Check if you have granted any third-party applications or services access to your GitHub account using a Personal Access Token (PAT). Sometimes, these applications create repositories on your behalf. You can revoke access for these applications from your GitHub account settings.

# If this works, please mark as answer. I need that Galaxy Brain Badge 🥲

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @letheriver2007 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
